symbulate/results.py

```diff
@@ -286,22 +287,46 @@ class RVResults(Results):
 
     def plot(self, type=None, alpha=None, normalize=True, jitter=False, 
         bins=30, **kwargs):
+        if type is None:
+            pass
+        elif isinstance(type, str):
+            type = (type,)
+        elif not isinstance(type, (tuple, list)):
+            raise Exception("I don't know how to plot a " + str(type))
+            
         
         dim = get_dimension(self)
         if dim == 1:
             counts = self._get_counts()
             if type is None:
                 heights = counts.values()
                 if sum([(i > 1) for i in heights]) > .8 * len(heights):
-                    type = "impulse"
+                    type = ("impulse",)
                 else:
-                    type = "bar"
-            if type == "bar":
-                if alpha is None:
-                    alpha = .5
-                plt.hist(self, normed=normalize, alpha=alpha, bins=bins, **kwargs)
-                plt.ylabel("Density" if normalize else "Count")
-            elif type == "impulse":
+                    type = ("hist",)
+            if alpha is None:
+                alpha = .5
+            if all(x in ("hist",) for x in type):
```

Comment:
This line is odd. First, why isn't it just `x == "hist"` instead of `x in ("hist",)`. Second, the line suggests that the user would specify something like `type=("hist", "hist", "hist")`. Why would they do that?